src/pages/__tests__/test.default.js

```diff
@@ -3,7 +3,7 @@ import Default from 'components/MicroComp/default'
 import { shallow } from 'enzyme'
 import schema2 from 'components/Schema/basicschema2'
```

Comment:
Same idea here. Data for the tests should be defined within the test as a `const`. I would envision the basicschema files being deleted after we move these components into production.